docs/reference/commandline/swarm_leave.md

```diff
@@ -31,12 +38,19 @@ dkp8vy1dq1kxleu9g4u78tlag    worker1   Ready   Active        Reachable
 dvfxp4zseq4s0rih1selh0d20 *  manager1  Ready   Active        Leader
 ```
 
-On a worker node, worker2 in the following example:
+To remove `worker2`, issue the following command from `worker2` itself:
 ```bash
 $ docker swarm leave
 Node left the default swarm.
 ```
 
+To remove an inactive node from the manager, use `docker node rm` instead. If the
+node is active and you need to remove it, add the `--force` flag. Depending on the
+type of task being run, this may cause transient disruptions.
```

Comment:
"To remove an inactive node from the node list, use `docker node rm` on a manager node."

I'm not sure we should talk about `rm`'s `--force` flag here. That's a way to kick a node out of a swarm, from a manager. Generally users would want to use `docker swarm leave` instead.

This whole section should be moved underneath the "On a manager node:" example below, which shows output that would be changed by running `docker node rm worker2`.